SampleOrigin/project.json

```diff
@@ -0,0 +1,50 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.1.0-*",
+      "type": "platform"
+    },
+    "Microsoft.AspNetCore.Diagnostics": "1.2.0-*",
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.2.0-*",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.2.0-*",
+    "Microsoft.Extensions.Logging.Console": "1.2.0-*",
+    "Microsoft.AspNetCore.StaticFiles": "1.2.0-*",
```

Comment:
I need it to launch the Index.html page in my origin to send CORS requests when I run SampleOrigin. Is there another way to launch the html page?